### PR TITLE
chore: add documentation on the binding mqtt based on the pubsub mqtt…

### PR DESCRIPTION
**Pull request by Hugome**
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

## Description

Document new MQTT binding options, based on the pubsub MQTT options/documentation

### CODE REVIEW

**Comment by orizohar** [comment]
@yaron2 - is this ok to go in to v1.0 or is this an addition in v1.1?

**Comment by artursouza** [comment]
> @yaron2 - is this ok to go in to v1.0 or is this an addition in v1.1?

This is v1.1

**Comment by orizohar** [comment]
> > @yaron2 - is this ok to go in to v1.0 or is this an addition in v1.1?
> 
> This is v1.1

@Hugome can you please rebase to v1.1?

**Review by orizohar — DISMISSED** [review]
Please rebase to v1.1

**Comment by Hugome** [comment]
Rebase done and base branch switch to v1.1

**Comment by artursouza** [comment]
The PR for this feature has not been merged yet, please, move this to 1.2.

https://github.com/dapr/components-contrib/pull/771

File: daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md [hunk 1]
```diff
@@ -21,20 +21,97 @@ spec:
   version: v1
   metadata:
   - name: url
-    value: mqtt[s]://[username][:password]@host.domain[:port]
+    value: "tcp://[username][:password]@host.domain[:port]"
```
Comment (orizohar) [inline daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md:24]: @Hugome - is this a breaking change? does the component now expect a different URL string?

Comment (Hugome) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md:24]: In the old component the scheme was ignored and forced to `tcp`
https://github.com/dapr/components-contrib/blob/a43a0d908bd8f32e0ea4b24607fd2ab2c4902ec5/bindings/mqtt/mqtt.go#L127

To ensure backward compatibly we can add a little `if scheme == mqtt => scheme = tcp` and `if scheme == mqtts => scheme = ssl` ?
(Same discussion with @pkedy on https://github.com/dapr/components-contrib/pull/771#discussion_r606337985)

Comment (artursouza) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md:24]: @Hugome Please, keep backwards compatibility so people don't need to change their current component to upgrade.

Comment (AaronCrawfis) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md:24]: @artursouza @Hugome is this resolved?

Comment (pkedy) [reply daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mqtt.md:24]: @AaronCrawfis I just committed some code that takes `mqtt`, `mqtts`, and `tcps` and converts them to the correct schemes that work with the MQTT client and preserve backward compatibility. IMO this PR _should_ update the docs for both the binding and pubsub to prefer `tcp` and `ssl` as the schemes.

See https://github.com/dapr/components-contrib/pull/771#discussion_r608196491

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**